### PR TITLE
Initial lonboard notebook

### PR DESCRIPTION
**Pull request by kylebarron**
I'm not too familiar with the data, but I was able to get it to render pretty easily once the job ran. I haven't looked at recreating the colormap, but that should be straightforward too

image

### CODE REVIEW

File: gedi-subset/notebooks/lonboard_exploration.ipynb [hunk 1]
```diff
@@ -0,0 +1,852 @@
+{
```
Comment (wildintellect) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Ah change the limit to 100 to get a much larger dataset for the purposes of testing this use case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (wildintellect) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>What command did you use to install <code>lonboard</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>I think we should clear all of the outputs from this notebook. Alternatively, at least clear the cells with user- and path-specific outputs, along with the output from the backoff library that's polling the job.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Let's remove this hard-coded assignment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Perhaps you can revert the changes to this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Perhaps eliminate the following few cells, which simply duplicate some lines from the preceding cell, and change <code>print(gedi_gdf.head())</code> to <code>display(gedi_gdf)</code> in the preceding cell as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Let's combine the 2 imports into 1 cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Perhaps the cells above, following the import cell, can be combined?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (kylebarron) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p><code>!pip install lonboard</code> from inside the notebook (and then reload the page to ensure the JS from the package is loaded onto the jupyter instance correctly) </p>

Comment (kylebarron) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>I didn't think I made any changes to this cell from the upstream notebook</p>

Comment (kylebarron) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: Ah thanks, I'll try that tomorrow

**Comment by kylebarron** [comment]
I updated the notebook to use @chuckwondo's existing gedi subsetter output, to hopefully be more reproducible and not require users to run a new gedi subsetting job just to check out this notebook. What do you think?

File: gedi-subset/notebooks/lonboard_exploration.ipynb [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
```
Comment (wildintellect) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Explain what lonboard is and link to it's website.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (wildintellect) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>nit "map_" is an odd variable name, e.g. in folium and ipyleaflet they alway just use "m"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (wildintellect) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>need to use absolute path <code>/projects/shared-buckets/dschuck/gedi-subset-examples/GNQ-L4A-subset2.gpkg</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (kylebarron) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: ok I'll use `m` instead. Usually I don't like single-character variable names that aren't obvious what they mean. And `map` overwrites the `map` stdlib function, hence `map_`

Comment (wildintellect) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: Agreed single letter variables are terrible, only suggested because it's what all the other libs use.

**Comment by kylebarron** [comment]
I think this should be good for re-review

File: gedi-subset/notebooks/lonboard_exploration.ipynb [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
```
Comment (wildintellect) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>nit: restart -&gt; refresh?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (wildintellect) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Oh Chrome and Firefox both call it Reload</p><p> </p>

Comment (wildintellect) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>Mostly I'm worried people will close their browser completely (not sure if that's a problem), but I didn't need to do that.</p>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>nit: wording seems a bit awkward/off: "its goal is to show that for data visualization". What is "that" referring to? Or did you mean something like, "use it," rather than, "show that"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (chuckwondo) [inline gedi-subset/notebooks/lonboard_exploration.ipynb:1]: <p>consider following the advice in the deprecation warning so that you can eliminate the warning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation-examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (kylebarron) [reply gedi-subset/notebooks/lonboard_exploration.ipynb:1]: I just removed that clause

**Review by chuckwondo — APPROVED** [review]
Fabulous!
